src/web/mjs/connectors/MangaParkEN.mjs

```diff
@@ -21,71 +21,80 @@ export default class MangaParkEN extends Connector {
 
     async _getMangas() {
         let mangaList = [];
-        const uri = new URL('/ajax.reader.home.release', this.url);
+        const uri = new URL('/browse?sort=name', this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'nav.d-md-block ul.pagination li:nth-last-child(2) a');
+        const pageCount = parseInt(data[0].text.trim());
+        for(let page = 1; page <= pageCount; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const uri = new URL('/browse?sort=name&page=' + page, this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, '#subject-list div.item');
+        return data.map( element => {
+            const a = element.querySelector('a.fw-bold');
+            this.cfMailDecrypt(a);
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(a, request.url),
```

Comment:
May extract the number directly from `a.pathname` since the full link is not really required

Reply:
Fixed